pkgs/tools/misc/pmbootstrap/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, git, openssl, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "pmbootstrap";
+  version = "1.30.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-uHN3nplQOMuBeQIxAocCVqwnmJUQZL67+iXLhQ7onps=";
+  };
+
+  buildInputs = [ ];
+
+  propagatedBuildInputs = [ git openssl ];
```

Comment:
Do we need binaries from those packages? If so please use makeWrapper. Otherwise move them to buildInputs if they are linked against. Or are those pythonPackages?

Reply:
Done